python/pyspark/sql/catalog.py

```diff
@@ -138,26 +138,6 @@ def listColumns(self, tableName, dbName=None):
                 isBucket=jcolumn.isBucket()))
         return columns
 
-    @since(2.0)
-    def createExternalTable(self, tableName, path=None, source=None, schema=None, **options):
```

Comment:
Maybe @gatorsmile or @cloud-fan want to comment on removing the `createExternalTable` methods

Reply:
let's remove it